wsutil/reader.go

```diff
@@ -12,6 +12,8 @@ import (
 // preceding NextFrame() call.
 var ErrNoFrameAdvance = errors.New("no frame advance")
 
+var ErrReadLimit = errors.New("read limit exceeded")
```

Comment:
Can you add a comment here? So `golint` will not yell at this line.

Reply:
I would suggest to rename the error to `ErrFrameTooLarge` to be more connected in the Reader.MaxFrameSize.